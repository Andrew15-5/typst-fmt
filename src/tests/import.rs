use super::*;

make_wide_test!(import_items_one_line_wide, IMPORT_ITEMS_ONE_LINE);
make_test!(import_items_one_line_narrow, IMPORT_ITEMS_ONE_LINE);

make_wide_test!(import_items_multiline_wide, IMPORT_ITEMS);
make_test!(import_items_multiline_narrow, IMPORT_ITEMS);

const IMPORT_ITEMS_ONE_LINE: &str = r#"
#import touying: touying-slide-wrapper, components, utils, config-common, config-page, touying-slide
#import touying: touying-slide-wrapper, components, utils, config-common, config-page, touying-slide,
#import touying: (touying-slide-wrapper, components, utils, config-common, config-page, touying-slide)
#import touying: (touying-slide-wrapper, components, utils, config-common, config-page, touying-slide,)
#import touying: (touying-slide-wrapper, components, utils, config-common, config-page, touying-slide )
#import touying: (touying-slide-wrapper, components, utils, config-common, config-page, touying-slide, )
"#;

const IMPORT_ITEMS: &str = r#"
#import touying: (
  touying-slide-wrapper,
  components,
  utils,
  config-common,
  config-page,
  touying-slide,
)
#import touying: (
  touying-slide-wrapper,
  components,
  utils,
  config-common,
  config-page,
  touying-slide
)
#import touying: ( touying-slide-wrapper,
  components,
  utils,
  config-common,
  config-page,
  touying-slide,

)
#import touying: (touying-slide-wrapper,
  components,
  utils,
  config-common,
  config-page,
  touying-slide)
"#;
