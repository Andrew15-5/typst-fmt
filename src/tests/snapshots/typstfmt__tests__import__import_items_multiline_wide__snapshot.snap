---
source: src/tests/import.rs
description: "INPUT\n===\n\"\\n#import touying: (\\n  touying-slide-wrapper,\\n  components,\\n  utils,\\n  config-common,\\n  config-page,\\n  touying-slide,\\n)\\n#import touying: (\\n  touying-slide-wrapper,\\n  components,\\n  utils,\\n  config-common,\\n  config-page,\\n  touying-slide\\n)\\n#import touying: ( touying-slide-wrapper,\\n  components,\\n  utils,\\n  config-common,\\n  config-page,\\n  touying-slide,\\n\\n)\\n#import touying: (touying-slide-wrapper,\\n  components,\\n  utils,\\n  config-common,\\n  config-page,\\n  touying-slide)\\n\"\n===\n\n#import touying: (\n  touying-slide-wrapper,\n  components,\n  utils,\n  config-common,\n  config-page,\n  touying-slide,\n)\n#import touying: (\n  touying-slide-wrapper,\n  components,\n  utils,\n  config-common,\n  config-page,\n  touying-slide\n)\n#import touying: ( touying-slide-wrapper,\n  components,\n  utils,\n  config-common,\n  config-page,\n  touying-slide,\n\n)\n#import touying: (touying-slide-wrapper,\n  components,\n  utils,\n  config-common,\n  config-page,\n  touying-slide)\n\n===\nFORMATTED\n===\n\n#import touying: touying-slide-wrapper, components, utils, config-common, config-page, touying-slide\n#import touying: touying-slide-wrapper, components, utils, config-common, config-page, touying-slide\n#import touying: touying-slide-wrapper, components, utils, config-common, config-page, touying-slide\n#import touying: touying-slide-wrapper, components, utils, config-common, config-page, touying-slide\n"
expression: formatted
---
"\n#import touying: touying-slide-wrapper, components, utils, config-common, config-page, touying-slide\n#import touying: touying-slide-wrapper, components, utils, config-common, config-page, touying-slide\n#import touying: touying-slide-wrapper, components, utils, config-common, config-page, touying-slide\n#import touying: touying-slide-wrapper, components, utils, config-common, config-page, touying-slide\n"
